src/configuration/prima.rs

```diff
@@ -72,14 +72,26 @@ impl Configuration for PrimaConfiguration {
 
 /// Represent an environment in which the datadog client runs.
 /// This is useful for enforcing rules based on environment for every application that uses the library.
-#[derive(PartialEq, Eq, Debug)]
+#[derive(PartialEq, Eq, Debug, Clone, Copy)]
 pub enum Environment {
     Dev,
     Qa,
     Staging,
     Production,
 }
 
+impl Environment {
+    /// Returns the string representation of the environment.
+    pub fn as_str(&self) -> &'static str {
```

Comment:
What about implementing `AsRef<str>`?

Reply:
I thought about it, but it would allow you to do stuff like this:

```rust
fn foo(str: &str) { ... }

foo(&Country::Uk);
```

Which imo I'd rather it be explicit instead:

```rust
foo(Country::Uk.as_str());
```